covid19.datacommons.io/portal/gitops.json

```diff
@@ -142,131 +148,49 @@
         "chartType": "count",
         "title": "Projects"
       },
-      "country_region": {
+      "symptoms": {
         "chartType": "bar",
-        "title": "Country Region"
+        "title": "Symptoms"
       },
-      "province_state": {
-        "chartType": "bar",
-        "title": "Province State"
+      "source": {
+        "chartType": "pie",
+        "title": "Source"
       }
     },
     "filters": {
       "tabs": [
         {
-          "title": "Location",
+          "title": "subject",
           "fields":[
-            "project_id",
-            "country_region",
-            "province_state"
+            "symptoms",
+            "country",
+            "visiting_wuhan",
+            "from_wuhan",
+            "death",
+            "recovered",
+            "reporting_date",
+            "source",
+            "age"
           ]
         }
       ]
     },
     "table": {
-      "enabled": false
+      "enabled": true
     },
     "buttons": [
-      {
-        "enabled": true,
-        "type": "manifest",
-        "title": "Download Manifest",
-        "leftIcon": "datafile",
-        "rightIcon": "download",
-        "fileName": "manifest.json",
-        "dropdownId": "download"
-      },
-      {
-        "enabled": true,
-        "type": "export-to-workspace",
-        "title": "Export To Workspace",
-        "leftIcon": "datafile",
-        "rightIcon": "download"
-      }
     ],
     "guppyConfig": {
-      "dataType": "location",
-      "nodeCountTitle": "Locations",
+      "dataType": "subject",
+      "nodeCountTitle": "Subjects",
       "manifestMapping": {
         "resourceIndexType": "file",
         "resourceIdField": "object_id",
-        "referenceIdFieldInResourceIndex": "location_id",
+        "referenceIdFieldInResourceIndex": "subject_id",
         "referenceIdFieldInDataIndex": "node_id"
       }
     }
   },
-  "fileExplorerConfig": {
```

Comment:
@mfshao don't we need to keep it to avoid some kind of issue when it's not configured? Can't remember the details

Reply:
> @mfshao don't we need to keep it to avoid some kind of issue when it's not configured? Can't remember the details

this is fine since we only need the data explorer but not the file explorer. Can't do it if it is vice versa since the FE depends on DE